vagrant_config.json

```diff
@@ -113,6 +113,19 @@
         "members": ["vagrant"], // to append this member to "docker" group
         "action": "create" // one of create, delete. Default: create
       },
+      "docker_registry": {
+        // set true to force re-login with all the defined entries, default: false (login once)
+        "force": false,
+        "entries": [{
```

Comment:
I'm not sure about these default entries. @phuonglm @datphan please consider this, why default here?